scripts/src/init.sh

```diff
@@ -56,7 +56,7 @@ cargo_init() {
   if [ -z $CI ]; then
     cargo install cargo-hack
     cargo install cargo-nextest
-  elif [ "$OSTYPE" == "linux-gnu" ]; then
+  elif [ "$OSTYPE" = "linux-gnu" ]; then
```

Comment:
Please check it twice

Reply:
We are because after Substrate update `protoc` is required to be installed

Reply:
I'm trying and everything is successful even in PR CI check, but not in master. There is an another try